### DISCUSSION TITLE
Is there a way to change Android navigationBarColor

### DISCUSSION THREAD

**Discussion by joesdu** [CommunityToolkit/Maui#651] [Q&A] [answered]
I want to change the navigationBarColor of Android platform to transparent, I have seen [Google Solutions](https://developer.android.google.cn/codelabs/gesture-navigation#3) but I don't know how to handle it in MAUI
![image](https://user-images.githubusercontent.com/13188169/191916514-574c400a-e0ae-43da-8027-4ea741500bc4.png)

**Comment by pictos** [comment, marked as answer]
There's a way to da that, but we didn't port to MCT yet. You can find the code here and implement it on your project (I believe it will work)

- https://github.com/xamarin/XamarinCommunityToolkit/blob/main/src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/AndroidSpecific/NavigationBar/PlatformNavigationBarEffect.android.cs

**Reply by joesdu** [reply]
Thanks, I'll be testing them later

**Reply by joesdu** [reply]
It's very awkward, as I'm learning mobile application development for the first time, I still don't know what to do after reading the code 😅. I'll wait for the CommunityToolKit migration and use it again.Thank you very much for your reply!

> There's a way to da that, but we didn't port to MCT yet. You can find the code here and implement it on your project (I believe it will work)
> 
> * https://github.com/xamarin/XamarinCommunityToolkit/blob/main/src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/AndroidSpecific/NavigationBar/PlatformNavigationBarEffect.android.cs

**Comment by CliffAgius** [comment]
Closed as answered
